walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -11,13 +11,64 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
 package com.google.sps;
 
-import java.util.Collection;
-
-public final class FindMeetingQuery {
+import java.util.*;
+ 
+public final class FindMeetingQuery { 
+  private ArrayList<TimeRange> viableTimeSlots = new ArrayList<TimeRange>(); 
+ 
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    ArrayList<TimeRange> busyTimes = new ArrayList<TimeRange>();
```

Comment:
Change this name to something more specific, otherwise the difference between this and allBusyTimes is not obvious.